### ISSUE TITLE
A11y_SQL Azure Data Studio_Import Extension - Specify Input File_ScreenReader: The button "browse"  is announced as table on tab key navigation.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#12961]
[“Check out Accessibility Insights](https://accessibilityinsights.io/)! – Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudio_Oct2020;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG1.3.1;#Benchmark;#NVDA;#AILimited;#AINotInScope;

**Environment Details**:
Application Name: SQL Azure Data Studio
Version 1.22.1 User Setup
Windows Version: Windows10
Screen reader: NVDA2020.3

**Repro Steps:**
1. Launch Azure Data Studio Insiders application.
2. Tab to server drop down and tab again to button new connection press enter key and add the server details.
3. Tab to the connected server and press enter key, use down key and select any database keep_MDW Right click on/ open the context menu for a database and select Data-tier application wizard from the context menu
4. Launch screen reader as NVDA and tab to button "browse"

**Actual:**
On navigating to the button "browse" using tab key screen reader announces it as a table.

**Issue not observed with JAWS**

**Same issue observed in the page:**

1. Import Extension - Preview Data

**Expected:**
The button browse should not be announced as a table.

**User Impact:**
The screen reader users will receive wrong information of control present on screen.

**Attachment:**
[#12961_A11y_SQL Azure Data Studio_Import Extension - Specify Input File_ScreenReader.zip](https://github.com/microsoft/azuredatastudio/files/5401642/12961_A11y_SQL.Azure.Data.Studio_Import.Extension.-.Specify.Input.File_ScreenReader.zip)

**Comment by aasimkhan30** [comment]
Closing this issue because of no repro.

@fsteffi this seems to be a screen reader bug present in earlier versions of windows 10. I have updated windows 10 to version 2004 and cannot repro this bug. Also, mac does not have this bug.

Please feel free to reopen this issue if you can repro it.

**Comment by fsteffi** [comment]
hi @alanrenmsft  has the item "Data-tier application wizard " been removed
![image](https://user-images.githubusercontent.com/24580570/220043713-4cd8ddb5-2295-49e4-950f-dd282d1a9ada.png)

GithubTags:#Regressed:02-20-23;

**Comment by fsteffi** [comment]
issue reproduces on
Version: 1.42.0-insider
Commit: 872095ed849537d6c96373eedcd187b3f707c460
Date: 2023-02-23T06:29:37.626Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.25300
[Screen Recording.webm](https://user-images.githubusercontent.com/24580570/220856881-67ce8263-9a0e-494d-b03a-3f27ac112e40.webm)

**Comment by aasimkhan30** [comment]
@fsteffi , can you please retest this issue and close it? It is fixed in the latest insider.

**Comment by BhanuPrakashGit** [comment]
**GithubTag:**#Regressed:05-26-23;#Closed;
Verified the bug in the environment: 
Version: 1.45.0-insider (user setup)
Commit: e62486bee6217e9749e687344d7beb1a7dbf0364
Date: 2023-05-25T05:30:58.800Z
VS Code: 1.70.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621

Bug is fixed and does not repro. Hence closing this bug.
PFA for reference.
[12961_Closed.webm](https://github.com/microsoft/azuredatastudio/assets/104351566/c8944335-5094-4070-bfb2-31387073cdf1)
